.ci/run_tests_minikube_linux.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# NOTE: This script assumes that minikube is installed and running, and using the docker driver on Linux
+# Due to networking issues with the docker driver and ingress on macOS/Windows, this script must be run on Linux
+
+# Share docker env with Minikube
+eval $(minikube docker-env)
+
+# error on unset variables
+set -u
+# print each command before executing it
+set -x
```

Comment:
Should we also add `set -e` to ensure the script can exit immediately if the command fails?

Reply:
I left it out since it would prevent us from debugging via kubectl any failed deployments with helm.